seqr/views/apis/staff_api_tests.py

```diff
@@ -0,0 +1,142 @@
+   u'disk.avail': None,
+   u'disk.percent': None
+  }]
+
+MISSED_INDEX="test_index_old"
+
+ES_CAT_INDICES=[{
+    "index": "test_index",
+    "docs.count": "122674997",
+    "store.size": "14.9gb",
+    "creation.date.string": "2019-11-04T19:33:47.522Z"
+  },
+  {
+    "index": "test_index_second",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  },
+  {
+    "index": "1kg_test_index",
+    "docs.count": "672312",
+    "store.size": "233.4mb",
+    "creation.date.string": "2019-10-03T19:53:53.846Z"
+  }]
+
+ES_CAT_ALIAS=[
+  {
+    "alias": "010203d4c1452eef3f8725bfdee23476",
+    "index": "test_index"
+  },
+  {
+    "alias": "1531a081753f4a135eef25b3496316dd",
+    "index": "1kg_test_index"
+  }]
+
+TEST_INDEX_SRC_PATH="test_index_src_path"
+TEST_INDEX_SECOND_SRC_PATH="test_index_second_src_path"
+TEST_INDEX_1KG_SRC_PATH="test_index_1kg_src_path"
+ES_INDEX_MAPPING={
+  "test_index": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "genomeVersion": "38",
+          "sampleType": "WES",
+          "sourceFilePath": TEST_INDEX_SRC_PATH,
+        },
+        "_all": {
+          "enabled": False
+        }
+      }
+    }
+  },
+  "test_index_second": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "25",
+          "hail_version": "0.2.24",
+          "genomeVersion": "38",
+         "sampleType": "WGS",
+          "sourceFilePath": TEST_INDEX_SECOND_SRC_PATH,
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  },
+  "1kg_test_index": {
+    "mappings": {
+      "variant": {
+        "_meta": {
+          "gencodeVersion": "19",
+          "genomeVersion": "37",
+          "sampleType": "WES",
+          "datasetType": "VARIANTS",
+          "sourceFilePath": TEST_INDEX_1KG_SRC_PATH
+        },
+        "_all": {
+          "enabled": False
+        },
+      }
+    }
+  }
+}
+
+
+class StaffAPITest(TestCase):
+    fixtures = ['users', '1kg_project']
+    multi_db = True
+
+    @mock.patch('elasticsearch_dsl.index.Index.get_mapping')
+    @mock.patch('elasticsearch.Elasticsearch')
+    def test_elasticsearch_status(self, mock_elasticsearch, mock_get_mapping):
+        url = reverse(elasticsearch_status)
+        _check_login(self, url)
+
+        mock_es_client = mock_elasticsearch.return_value
+        mock_es_client.cat.allocation.return_value = ES_CAT_ALLOCATION
+        mock_es_client.cat.indices.return_value = ES_CAT_INDICES
+        mock_es_client.cat.aliases.return_value = ES_CAT_ALIAS
+        mock_get_mapping.return_value = ES_INDEX_MAPPING
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+        response_json = response.json()
+        self.assertListEqual(response_json.keys(), ['indices', 'errors', 'diskStats', 'elasticsearchHost'])
+        for index in response_json['indices']:
```

Comment:
You should create objects that contain what you expect the index dictionary to have and test against that.  You don;t really need to test that every index is as expected, I would recommend testing 1 that has projects and 1 that doesn't. So for instance, the tests you would have here are 
```
self.assertEqual(len(response_json['indices']), 3)
self.assertDictEqual(response_json['indices'][0], EXPECTED_TEST_INDEX)
self.assertDictEqual(response_json['indices'][2], EXPECTED_1KG_INDEX)
```

Reply:
Good. I will do as you've suggested.

Reply:
I also reformatted the entire file. So there are manage changes. But you can just review the response data checking part and those expected data.